### PR TITLE
(FM-7785) dont use default function in spec/acceptance

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: spec/acceptance/firewall_attributes_exceptions_spec.rb [hunk 1]
```diff
@@ -405,7 +405,7 @@ class { '::firewall': }
 
   # iptables version 1.3.5 is not suppored by the ip6tables provider
   # iptables version 1.4.7 fails for multiple hl entries
-  if default['platform'] !~ %r{(el-5|el-6|sles-10|sles-11)}
+  describe 'testing ipv6', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) || (os[:family] == 'sles') do
```
Comment (david22swan) [inline spec/acceptance/firewall_attributes_exceptions_spec.rb]: You can just do "os[:release].start_with?('5', '6')".

File: spec/acceptance/firewall_attributes_exceptions_spec.rb [hunk 2]
```diff
@@ -1130,22 +1126,21 @@ class { '::firewall': }
...
+            shell('ip6tables-save') do |r|
+              expect(r.stdout).not_to match(%r{-A INPUT -p tcp -m addrtype\s.*\sLOCAL -m addrtype\s.*\sLOCAL -m comment --comment "619 - test" -j ACCEPT})
             end
           end
+        end
 
-          context 'when multiple addrtype', unless: (os[:family] == 'redhat' && os[:release].start_with?('5')
-                                                    ) do
-            pp105 = <<-PUPPETCODE
+        context 'when multiple addrtype', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) do
```
Comment (david22swan) [inline spec/acceptance/firewall_attributes_exceptions_spec.rb]: Same as above

File: spec/acceptance/firewall_iptmodules_spec.rb [hunk 3]
```diff
@@ -60,17 +60,15 @@ class { '::firewall': }
...
       end
     end
   end
 
-  # iptables version 1.3.5 is not suppored by the ip6tables provider
-  if default['platform'] =~ %r{ubuntu-14\.04}
-    describe 'ip6tables ipt_modules tests' do
-      context 'when all the modules with multiple args' do
-        pp3 = <<-PUPPETCODE
+  describe 'ip6tables ipt_modules tests', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) || (os[:family] == 'sles') do
```
Comment (david22swan) [inline spec/acceptance/firewall_iptmodules_spec.rb]: Same as above

File: spec/acceptance/firewall_tee_spec.rb [hunk 4]
```diff
@@ -1,59 +1,56 @@
 require 'spec_helper_acceptance'
 
-describe 'firewall tee' do
+describe 'firewall tee', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) || (os[:family] == 'sles') do
```
Comment (david22swan) [inline spec/acceptance/firewall_tee_spec.rb]: Same as above

File: spec/acceptance/firewall_time_spec.rb [hunk 5]
```diff
@@ -1,71 +1,73 @@
 require 'spec_helper_acceptance'
 
-describe 'firewall time' do
+describe 'firewall time', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) || (os[:family] == 'sles') do
```
Comment (david22swan) [inline spec/acceptance/firewall_time_spec.rb]: Same as above

**Comment by david22swan** [comment]
@tphoney Some small notes sorry.

**Comment by tphoney** [comment]
not that small @david22swan, i can still read them :P

File: spec/acceptance/firewall_attributes_exceptions_spec.rb (hunk 1 above)
Comment (tphoney) [reply spec/acceptance/firewall_attributes_exceptions_spec.rb]: nice catch, i wasnt aware that start_with accepted multiple prefixes.

image

**Comment by tphoney** [comment]
image
@david22swan should be good to go

**Pull request merged**